UnityProject/Assets/Editor/LevelBuilder/GUI/Menus/PickupMenu.cs

```diff
@@ -0,0 +1,47 @@
+using UnityEngine;
+using UnityEditor;
+
+public class PickupMenu : IMenu
+{
+
+    private GUIContent pickupPickerButtonTitle = new GUIContent("View Pickups");
+
+    public void Display()
```

Comment:
it displays and listens

Reply:
UnityEngine listens for `OnGUI()` which draws every frame using `Display()`, afraid there's no other way of splitting this